### PR TITLE
Install widget

### PR DESCRIPTION
**Pull request by seromenho**
- Add login command that stores auth_token locally
- Add logout command that only cleans the local auth_token
- Add command to install a widget.
- Add support for multiple environments

### CODE REVIEW

**Comment by squallstar** [comment]
Install to be renamed `publish`

File: index.js [hunk 1]
```diff
@@ -6,5 +6,8 @@ program
   .version('1.3.0')
   .command('create-widget [name]', 'create a new widget')
   .command('run-widget', 'run the current widget for development')
+  .command('install-widget', 'install the current widget for development')
```
Comment (squallstar) [inline index.js]: `publish the current widget on fliplet studio`

File: config/production.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "api_url": "http://api.fliplet.com/"
```
Comment (squallstar) [inline config/production.json]: https

File: config/staging.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "api_url": "http://staging.api.fliplet.com/"
```
Comment (squallstar) [inline config/staging.json]: https

File: lib/env.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+function getEnvironment(env) {
+  return (env === 'dev' || env === 'staging')
+    ? env
+    : 'production';
+}
+
+
+const ENV = require('../config/' + getEnvironment(envStored));
+
+console.log(ENV);
```
Comment (squallstar) [inline lib/env.js]: to be removed

File: lib/template.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 const Handlebars = require('handlebars');
 const request = require('request');
 
-const ENV = require('../config/env');
+const ENV = require('../lib/env').env;
```
Comment (squallstar) [inline lib/template.js]: `lib/env` is redundant with `env`, shall we call it `lib/config` ? (like on the fliplet-api)

Comment (seromenho) [inline lib/template.js]: I also make use of the function `getEnvironment` to get the correct environment

Comment (seromenho) [inline lib/template.js]: Ah, I see. Updating

**Comment by squallstar** [comment]
I also recommend you increasing the version on `package.json`, as it's required for publishing the package on npm.

**Pull request merged**